Gemfile

```diff
@@ -54,3 +54,6 @@ gem "tzinfo-data", platforms: [:mingw, :mswin, :x64_mingw, :jruby]
 
 # Use Bootstrap as the CSS library
 gem "bootstrap-sass", "3.3.6"
+
+# Use hash password
+gem "bcrypt", "3.1.11"
```

Comment:
"bcrypt" gem cung cấp cách thức để xử lý lưu password một cách an toàn, sử dụng thuật toán bcrypt đê băm password, chống người khác xâm nhập và copy database có thể lấy được password của người dùng